includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -572,34 +572,34 @@ static function ( $validation_error ) {
 			}
 		}
 
+		$attr_spec_list = array_merge(
```

Comment:
Previously this was assigning to `$merged_attr_spec_list`. However, the `$attr_spec_list` variable was no longer being used after this point anyway, so this just merges the global onto that existing variable and it is then used hereafter.